Dockerfile-frontend

```diff
@@ -22,7 +22,7 @@ COPY . .
 
 # install required dependencies which slim image doesn't have
 RUN apt-get update && \
-	apt-get install -y apt-transport-https procps curl wget git python2 make gcc g++ unzip && \
+	apt-get install -y iputils-ping iproute2 procps net-tools telnet apt-transport-https procps curl wget git python2 make gcc g++ unzip && \
```

Comment:
This will help to perform some basic troubleshoot check from connectivity purpose from inside the container. Also it was requested from one of our customer